packages/commonwealth/client/scripts/views/pages/NotificationSettings/index.tsx

```diff
@@ -0,0 +1,180 @@
+import { CommunityAlert, ThreadSubscription } from '@hicommonwealth/schemas';
+import { useFlag } from 'hooks/useFlag';
+import { useCommonNavigate } from 'navigation/helpers';
+import React, { useCallback, useState } from 'react';
+import app from 'state';
+import { trpc } from 'utils/trpcClient';
+import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
+import CWPageLayout from 'views/components/component_kit/new_designs/CWPageLayout';
+import {
+  CWTab,
+  CWTabsRow,
+} from 'views/components/component_kit/new_designs/CWTabs';
+import { CommunityEntry } from 'views/pages/NotificationSettings/CommunityEntry';
+import { getFirebaseMessagingToken } from 'views/pages/NotificationSettings/getFirebaseMessagingToken';
+import { useCommunityAlerts } from 'views/pages/NotificationSettings/useCommunityAlerts';
+import { useThreadSubscriptions } from 'views/pages/NotificationSettings/useThreadSubscriptions';
+import useNotificationSettings from 'views/pages/NotificationSettingsOld/useNotificationSettings';
+import { z } from 'zod';
+import { CWText } from '../../components/component_kit/cw_text';
+import { PageLoading } from '../loading';
+import { SubscriptionEntry } from './SubscriptionEntry';
+import './index.scss';
+
+type NotificationSection = 'community-alerts' | 'subscriptions';
+
+const NotificationSettings = () => {
+  const navigate = useCommonNavigate();
+  const threadSubscriptions = useThreadSubscriptions();
+  const communityAlerts = useCommunityAlerts();
+  const enableKnockPushNotifications = useFlag('knockPushNotifications');
+  const registerClientRegistrationToken =
+    trpc.subscription.registerClientRegistrationToken.useMutation();
+
+  const communityAlertsIndex = createIndexForCommunityAlerts(
+    (communityAlerts.data as unknown as ReadonlyArray<
+      z.infer<typeof CommunityAlert>
+    >) || [],
+  );
+
+  const { bundledSubs } = useNotificationSettings();
+
+  const [threadsFilter, setThreadsFilter] = useState<readonly number[]>([]);
+  const [section, setSection] =
+    useState<NotificationSection>('community-alerts');
+
+  const handleUnsubscribe = useCallback(
+    (id: number) => {
+      setThreadsFilter([...threadsFilter, id]);
+    },
+    [threadsFilter],
+  );
+
+  const handlePushNotificationSubscription = useCallback(() => {
+    async function doAsync() {
+      const permission = await Notification.requestPermission();
+      if (permission === 'granted') {
+        console.log('Notification permission granted.');
+        const token = await getFirebaseMessagingToken();
+        await registerClientRegistrationToken.mutateAsync({
+          id: 'none',
+          token,
+        });
+      }
+    }
+
+    doAsync().catch(console.error);
+  }, [registerClientRegistrationToken]);
+
+  if (threadSubscriptions.isLoading) {
+    return <PageLoading />;
+  } else if (!app.isLoggedIn()) {
+    navigate('/', { replace: true });
```

Comment:
we normally show `<PageNotFound/>` when user visits auth gated pages without any authentication.